script/files.lua

```diff
@@ -919,10 +919,10 @@ function m.resolvePathPlaceholders(path)
             if addonsPath then
                 return addonsPath
             end
-            local client = require 'client'
-            local storagePath = client.getOption('storagePath')
+            local client = require("client")
+            local storagePath = client.getOption("storagePath")
             if storagePath then
```

Comment:
Is this a suitable way to check for `client.getOption` actually providing me the `storagePath` or is it never falsy? I have no idea what [`nonil`](https://github.com/LuaLS/lua-language-server/blob/b815469b3df0655b59a8d2eae2770a3b85c98ad8/script/client.lua#L43-L45) will return for a basic lookup 😆.

Reply:
You should check nil